src/nuklear_text.c

```diff
@@ -34,8 +34,7 @@ nk_widget_text(struct nk_command_buffer *o, struct nk_rect b,
         label.w = NK_MAX(1, 2 * t->padding.x + (float)text_width);
         label.x = (b.x + t->padding.x + ((b.w - 2 * t->padding.x) - label.w) / 2);
         label.x = NK_MAX(b.x + t->padding.x, label.x);
-        label.w = NK_MIN(b.x + b.w, label.x + label.w);
-        if (label.w >= label.x) label.w -= label.x;
+        label.w = NK_MIN(b.w, label.w);
```

Comment:
This change looks unrelated?

Reply:
``` suggestion
        label.w = NK_MIN(b.x + b.w, label.x + label.w);
        if (label.w >= label.x) label.w -= label.x;
```

Reply:
`x+w` is the coordinate, `w` is the distance, and the physical meaning of associating these two quantities seems incorrect